### PR TITLE
[VEN-2285] Rewards Deployment Sepolia and Ethereum

### PR DESCRIPTION
**Pull request by 0xlucian**
## Description

This PR refactor the final config for the rewards on Sepolia and Ethereum and deploys all the Reward Distributors on both `Sepolia` and `Ethereum`

Resolves #VEN-2285

## Checklist

- [ ] I have updated the documentation to account for the changes in the code.
- [ ] If I added new functionality, I added tests covering it.
- [ ] If I fixed a bug, I added a test preventing this bug from silently reappearing again.
- [ ] My contribution follows [Venus contribution guidelines](docs/CONTRIBUTING.md).

### CODE REVIEW

File: helpers/deploymentConfig.ts [hunk 1]
```diff
@@ -2641,7 +2641,26 @@ export const globalConfig: NetworkConfig = {
...
+            markets: ["CRV"],
+            supplySpeeds: ["9259259260000"],
+            borrowSpeeds: ["13888888890000"],
+          },
+          {
+            asset: "XVS",
+            markets: ["crvUSD"],
+            supplySpeeds: ["37037037036000"],
+            borrowSpeeds: ["55555555554000"],
+          },
```
Comment (chechu) [inline helpers/deploymentConfig.ts:2656]: I think we could combine this in one RewardDistributor contract

File: helpers/deploymentConfig.ts [hunk 2]
```diff
@@ -2687,7 +2706,26 @@ export const globalConfig: NetworkConfig = {
...
+            markets: ["wstETH"],
+            supplySpeeds: ["222222222200000"],
+            borrowSpeeds: ["0"],
+          },
+          {
+            asset: "XVS",
+            markets: ["WETH"],
+            supplySpeeds: ["254629629600000"],
+            borrowSpeeds: ["594135802500000"],
+          },
```
Comment (chechu) [inline helpers/deploymentConfig.ts:2721]: I think we could combine this in one RewardDistributor contract

File: helpers/deploymentConfig.ts [hunk 3]
```diff
@@ -2641,7 +2641,26 @@ export const globalConfig: NetworkConfig = {
...
+            asset: "XVS",
+            markets: ["crvUSD"],
+            supplySpeeds: ["37037037036000"],
+            borrowSpeeds: ["55555555554000"],
+          },
+          {
+            asset: "CRV",
+            markets: ["crvUSD"],
+            supplySpeeds: ["77160493840000000"],
+            borrowSpeeds: ["115740740800000000"],
```
Comment (chechu) [inline helpers/deploymentConfig.ts:2661]: They should be divided by 100

File: helpers/deploymentConfig.ts [hunk 4]
```diff
@@ -2509,27 +2509,15 @@ export const globalConfig: NetworkConfig = {
         rewards: [
           {
             asset: "XVS",
-            markets: ["WBTC"],
-            supplySpeeds: ["190258751902587"], // 190258751902587 XVS over 365 days (2628000 blocks)
-            borrowSpeeds: ["190258751902587"], // 190258751902587 XVS over 365 days (2628000 blocks)
+            markets: ["vWETH_Core","vWBTC_Core, vUSDT_Core, vUSDC_Core","vcrvUSD_Core"],
```
Comment (chechu) [inline helpers/deploymentConfig.ts:2512]: Is this syntax correct? The second item of the array is the string `"vWBTC_Core, vUSDT_Core, vUSDC_Core"`

File: helpers/deploymentConfig.ts [hunk 5]
```diff
@@ -2509,27 +2509,15 @@ export const globalConfig: NetworkConfig = {
         rewards: [
           {
             asset: "XVS",
-            markets: ["WBTC"],
-            supplySpeeds: ["190258751902587"], // 190258751902587 XVS over 365 days (2628000 blocks)
-            borrowSpeeds: ["190258751902587"], // 190258751902587 XVS over 365 days (2628000 blocks)
+            markets: ["WETH","WBTC, USDT, USDC","crvUSD"],
```
Comment (chechu) [inline helpers/deploymentConfig.ts:2512]: Maybe something like this:
```
markets: ["WETH", "WBTC", "USDT", "USDC", "crvUSD"],
```

File: helpers/deploymentConfig.ts [hunk 6]
```diff
@@ -2509,27 +2509,15 @@ export const globalConfig: NetworkConfig = {
         rewards: [
           {
             asset: "XVS",
-            markets: ["WBTC"],
-            supplySpeeds: ["190258751902587"], // 190258751902587 XVS over 365 days (2628000 blocks)
-            borrowSpeeds: ["190258751902587"], // 190258751902587 XVS over 365 days (2628000 blocks)
+            markets: ["WETH","WBTC", "USDT", "USDC","crvUSD"],
+            supplySpeeds: [ "27777777777777","83333333333333","83333333333333","83333333333333","37037037037036"],
+            borrowSpeeds: ["41666666666666","124999999999999","125000000000000","125000000000000","55555555555555"],
```
Comment (chechu) [inline helpers/deploymentConfig.ts:2514]:
```suggestion
            borrowSpeeds: ["41666666666666","125000000000000","125000000000000","125000000000000","55555555555555"],
```

File: helpers/deploymentConfig.ts [hunk 7]
```diff
@@ -2509,27 +2509,15 @@ export const globalConfig: NetworkConfig = {
         rewards: [
           {
             asset: "XVS",
-            markets: ["WBTC"],
-            supplySpeeds: ["190258751902587"], // 190258751902587 XVS over 365 days (2628000 blocks)
-            borrowSpeeds: ["190258751902587"], // 190258751902587 XVS over 365 days (2628000 blocks)
+            markets: ["WETH","WBTC", "USDT", "USDC","crvUSD"],
+            supplySpeeds: [ "27777777777777","83333333333333","83333333333333","83333333333333","37037037037036"],
+            borrowSpeeds: ["41666666666666","125000000000000","125000000000000","125000000000000","55555555555555"],
```
Comment (GitGuru7) [inline helpers/deploymentConfig.ts:2514]: Can we mention the number of tokens, duration and total blocks for rewards in comments as we are doing for other networks?

**Review by chechu — COMMENTED** [review]
LGTM

**Review by GitGuru7 — APPROVED** [review]

**Review by chechu — APPROVED** [review]

**Pull request merged**